nixos/modules/services/audio/mopidy.nix

```
@@ -83,6 +83,7 @@ in {
       description = "mopidy music player daemon";
       serviceConfig = {
         ExecStart = "${mopidyEnv}/bin/mopidy --config ${concatStringsSep ":" ([mopidyConf] ++ cfg.extraConfigFiles)}";
+        Restart = "always";
```

```suggestion
        Restart = "on-failure";
```
This is what's [recommended](https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#) upstream and helps prevent the service from restarting even when you are trying to stop it

---

Indeed, thank you. I've updated this.